frontend/src/screens/researcher/studies/create/edit-study.tsx

```diff
@@ -122,9 +123,12 @@ const FormContent: FC<{
     const [maxStep, setMaxStep] = useLocalstorageState<StudyStep>(`study-max-progress-${id}`, 0)
     const { isValid, isDirty } = useFormState()
 
-    if (!isDraft(study) && !isNew) {
-        return <Navigate to={`/study/overview/${id}`} />
-    }
+    useEffect(() => {
+        if (!isDraft(study) && !isNew) {
```

Comment:
There was some really weird inconsistent issue plaguing the playwright tests where the study would save, but it would redirect to the study overview page rather than the study creation flow... i believe this should finally fix that